### PR TITLE
Newbanner

### PR DESCRIPTION
**Pull request by Carreau**
FInish up #8773 implemented @fperez requests on top of @Achifaifa work.

```
~ $ ipython
Python 3.5.2 |Anaconda custom (x86_64)| (default, Jul  2 2016, 17:52:12)
Type 'copyright', 'credits' or 'license' for more information
IPython 5.0.0.dev -- An enhanced Interactive Python. Type '?' for help.

In [1]: ?

<open pager>

IPython -- An enhanced Interactive Python
=========================================

IPython offers a combination of convenient shell features, special commands
and a history mechanism for both input (command history) and output (results
caching, similar to Mathematica). It is intended to be a fully compatible
replacement for the standard Python interpreter, while offering vastly
improved functionality and flexibility.

At your system command line, type 'ipython -h' to see the command line
options available. This document only describes interactive features.

GETTING HELP
------------

Within IPython you have various way to access help:

  ?         -> Introduction and overview of IPython's features (this screen).

  %quickref -> Quick reference of all IPython specif syntax and magics.

  object?   -> Details about 'object', use 'object??' for extra details.

  help      -> Access Python's own help system.

If you are in terminal IPython you can quit this screen by pressing `q`.


MAIN FEATURES
-------------
```

### CODE REVIEW

**Comment by Achifaifa** [comment]
Nice to see this thing finally settled :)

File: IPython/core/usage.py [hunk 1]
```diff
@@ -76,6 +76,22 @@
 At your system command line, type 'ipython -h' to see the command line
 options available. This document only describes interactive features.
 
+GETTING HELP
+------------
+
+Within IPython you have various way to access help:
+
+  ?         -> Introduction and overview of IPython's features (this screen).
+
```
Comment (fperez) [inline IPython/core/usage.py]: I would remove these blank lines in between each one of these, like it used to be in the old banner.  I don't think we need all the extra whitespace here.

File: IPython/core/usage.py [hunk 2]
```diff
@@ -76,6 +76,22 @@
...
+  %quickref -> Quick reference of all IPython specif syntax and magics.
```
Comment (fperez) [inline IPython/core/usage.py]: typo: 'specific'

File: IPython/core/usage.py [hunk 3]
```diff
@@ -76,6 +76,22 @@
...
+
+  object?   -> Details about 'object', use 'object??' for extra details.
```
Comment (fperez) [inline IPython/core/usage.py]: I would move this one above so that '?' and 'foo?' are discussed next to each other.

Comment (fperez) [inline IPython/core/usage.py]: Perhaps also change "extra details" -> "more information", to avoid repeating the word details.

**Comment by fperez** [comment]
Looks good, though I'd like to squash this into just one or two commits... While we normally don't do a lot of PR squashing, this one has a *huge* amount of commits for just a couple of simple text changes.  This should be just one or two commits, no more.

Otherwise once we do the small fixes in the comments, great!

File: IPython/core/usage.py (hunk 3 above)
Comment (willingc) [inline IPython/core/usage.py]: Perhaps:

```
?                    -> Introduction and overview of IPython's features (this screen).
object?              -> Information about 'object'
object??             -> Provides more detailed, verbose information about 'object'
%quickref            -> Quick reference of IPython syntax and magics.
help                 -> Access the help system of the Python language
```

Comment (fperez) [inline IPython/core/usage.py]: Sounds good, I'd only adjust this:

On Fri, Aug 26, 2016 at 5:24 AM, Carol Willing <notifications@github.com>
wrote:

> object?? -> Provides more detailed, verbose information about 'object'


to simply "More detailed, ...". removing "provides": the others don't use
that verb, and it makes the line longer than 80 chars.

Thanks @willingc!


-- 
Fernando Perez (@fperez_org; http://fperez.org)
fperez.net-at-gmail: mailing lists only (I ignore this when swamped!)
fernando.perez-at-berkeley: contact me here for any direct mail

**Comment by Carreau** [comment]
> Looks good, though I'd like to squash this into just one or two commits...

Done, with Authorship of @Achifaifa conserved, rebased on master as well; and @willingc  and @fperez comments taken into account.

**Comment by willingc** [comment]
LGTM. Thanks @Achifaifa and @Carreau. Feel free to merge.

**Comment by fperez** [comment]
Great, thanks everyone!!

**Pull request merged**